omero/sysadmins/publication.rst

```diff
@@ -0,0 +1,103 @@
+Complete workflow for hosting data for a publication
+====================================================
+
+The following describes the steps of a complete workflow for using OMERO to
+host public data associated with a publication. It is illustrated using an
+example publication from our lab in Dundee,
+`Schleicher et al, 2017 <http://dx.doi.org/10.1098/rsob.170099>`_.
+
+Group setup
+-----------
+
+#. Create a dedicated :doc:`public group <public>` to host the raw data
```

Comment:
Sorry for the late nitpick, but reading this the second time, and navigating to the link "public group", I think this is a bit confusing. In fact, there is no concept of "public group" in OMERO.web. This is just a normal read-only group, of which the public user is a member. The doc link at present leads to a page where there is no word about "public group" either. 

So:

``Create a dedicated public group to host...`` -> ``Create a dedicated read-only group to host the raw data``

Alternatively, add a link to the place which explains how to create new groups in our doc, probably the "Creating groups and users" chapter of http://help.openmicroscopy.org/facility-manager.html, which, unfortunately, cannot be linked to directly.

Reply:
I'll use https://docs.openmicroscopy.org/omero/5.4.1/sysadmins/cli/usergroup.html for the link since this is in the sysadmin docs anyway